sympy/concrete/summations.py

```diff
@@ -492,6 +492,17 @@ def is_convergent(self):
             except NotImplementedError:
                 pass
 
+        ### ----------- Raabe's test ---------------- ###
+        if lim_evaluated == 1:
+            lim_val = Limit(sym * (abs(sequence_term)/abs(sequence_term).subs(sym, sym + 1))
```

Comment:
May be this term is not completely correct?

Reply:
This test is OK for absolute convergence. It is also valid for series with positive (or negative) terms (from some point on). Otherwise this test must not return False since the series can be convergent but not absolutely convergent.

Reply:
Under the condition `lim_evaluated == 1`, the terms have the same sign. Hence it should be possible to use it here.

Reply:
Correction: `lim_evaluated` is not what I thought is should be. It says nothing of the signs since it takes the absolute values.

Reply:
We should probably omit `abs` here.

Reply:
I think you are right. Though the point mentioned in the comment was the `-1` should be inside the bracket. The term `- S(1)` overjumps the bracket?
That is it should be
```
In [16]: Limit(n*(u_n/u_p - 1), n, oo)
Out[16]: 
    ⎛  ⎛       uₙ  ⎞⎞
lim ⎜n⋅⎜-1 + ──────⎟⎟
n─→∞⎝  ⎝     u₍ₙ₊₁₎⎠⎠

```

Currently it is
```
    ⎛ n⋅│uₙ│     ⎞
lim ⎜──────── - 1⎟
n─→∞⎝│u₍ₙ₊₁₎│    ⎠

```

Reply:
That's right, the closing parenthesis comes too early.